lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -753,10 +753,16 @@ def add_mc_directory_trans_id(xml, credit_card, three_d_secure)
         xml.tag!(:MCDirectoryTransID, three_d_secure[:ds_transaction_id]) if three_d_secure[:ds_transaction_id]
       end
 
-      def add_mc_ucafind(xml, credit_card, three_d_secure)
+      def add_mc_ucafind(xml, credit_card, three_d_secure, options)
         return unless three_d_secure
 
-        xml.tag! :UCAFInd, '4'
+        if options[:alternate_ucaf_flow]
```

Comment:
A few questions on this if else block. 

1. If `alternate_ucaf_flow = true` are we ok if `options[:ucaf] = nil`?
2. Why have we added the eci check? 
3. Without the `alternate_ucaf_flow` feature flag enabled (so in the else block), we are still allowing a merchant to set options[:ucaf] and forwarding that to the gateway. I don't have a problem with this, but now the `eci` isn't being checked like it is in the block above, which seems like the functionality differs a bit.

Reply:
If options[:ucaf] is nil then we won't send :UCAFInd. But looking at the ticket again it looks like I miss read it and we should send it as nil if not was provided

Reply:
To answer your third question, it was mentioned in the ticket that we should send the GSF if provided by the customer but it didn't mention eci check.

Reply:
Ah, I should have read the ticket. This all makes sense then! I'll approve with the current code. Thanks for explaining things to me.